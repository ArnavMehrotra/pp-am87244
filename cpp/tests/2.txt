A B C
1 3 5
2.5 4 7
22 30 50
