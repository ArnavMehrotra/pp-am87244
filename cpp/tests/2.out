A B C 
22 30 50 
